pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1320,12 +1326,11 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 			VCPUs.VCPU = append(VCPUs.VCPU, vcpu)
 		}
 		domain.Spec.VCPUs = VCPUs
-	}
-
-	domain.Spec.CPU.Topology = cpuTopology
-	domain.Spec.VCPU = &api.VCPU{
-		Placement: "static",
-		CPUs:      cpuCount,
```

Comment:
@enp0s3 instead of coping the vcpu struct, perhaps it's just better to rename the `cpuCount` variable name inside this `if` block? (starting from line 1308)

Reply:
Going forward, we should also come up with a function that would calculate the correct amount of enabled cpus, instead of relying on a single variable.

This code in `Convert_v1_VirtualMachineInstance_To_api_Domain` has become very long. We should break it.

Reply:
@vladikr Agree, we already had a discussion this morning about this, which outcome is to extract a method for it.
Also, we also think that future works/changes to this function should be made with the goal of refactoring this function.